polaris-react/src/components/ActionList/components/Section/Section.tsx

```diff
@@ -18,11 +17,13 @@ export interface SectionProps {
   actionRole?: 'option' | 'menuitem' | string;
   /** Callback when any item is clicked or keypressed */
   onActionAnyItem?: ActionListItemDescriptor['onAction'];
+  isFirst?: boolean;
```

Comment:
```suggestion
  /** Whether it is the first in a group of suggestions */
  isFirst?: boolean;
```

This may not be the best prop description, feel free to edit 😅 
Whatever it ends up being, it will probably require a re-run of the get-props script